src/_selectors/FullTradeSelectors.js

```diff
@@ -126,17 +126,40 @@ const durationSecHelper = duration => {
     return durationToSecs(d, u);
 };
 
+const shouldBlockExist = (min, max, unit) => {
```

Comment:
I suggest we do the extraction later ~
I'll move this function to private function and rename it so that full trade works better